---
float32[16] final_pose
string matched_model
